front/migrations/db/migration_93.sql

```diff
@@ -0,0 +1,14 @@
+-- Migration created on Sep 26, 2024
+ALTER TABLE "labs_transcripts_histories" ADD COLUMN "userId" INTEGER;
+
+UPDATE "labs_transcripts_histories"
```

Comment:
You also need to change the LabsTranscriptsHistoryModel.create to pass the `userId`.